packages/monaco/src/browser/protocol-to-monaco-converter.ts

```diff
@@ -131,7 +131,8 @@ export class ProtocolToMonacoConverter {
             startColumn: diagnostic.range.start.character + 1,
             endLineNumber: diagnostic.range.end.line + 1,
             endColumn: diagnostic.range.end.character + 1,
-            relatedInformation: this.asRelatedInformations(diagnostic.relatedInformation)
+            relatedInformation: this.asRelatedInformations(diagnostic.relatedInformation),
+            tags: diagnostic.tags
```

Comment:
Looks like we did not handle tags _at all_ before. Is there an instance of a "unnecessary" tag we could use to test?

Reply:
I think @alvsan09 tested it already in his comment. See the `myUndefined` variable and its 'fading' effect:

![image](https://user-images.githubusercontent.com/76971376/125120242-32780080-e0c0-11eb-9dfc-8ef3439d819a.png)